include/z64light.h

```diff
@@ -4,6 +4,7 @@
 #include "ultra64.h"
 #include "PR/gbi.h"
 #include "color.h"
+#include "z64math.h"
```

Comment:
Why was this added?

Reply:
Because this header rely on structs from `z64math.h`, like `Vec3f`. Currently it just happen to work because every time `z64light.h` is included `z64math.h` has already been included before